lib/organization.js

```diff
@@ -4,7 +4,13 @@ import { RESOURCE_BASE_URI, USERS_GRAPH, AUTH_ORG_CODE_CLAIM, AUTH_ORG_NAME_CLAI
 
 const ovoCodeFromString = function (s) {
   const match = s.match(/OVO\d{6}/);
-  return match ? match[0] : match;
+  // return match ? match[0] : match;
+  // Temporary fix. Non VO organizations don't send us an OVO-code, instead they
```

Comment:
Tbh I don't think this change will be temporary since not all organisations do have an OVO-code. I would just leave out this `ovoCodeFromString` helper as hotfix.